### PR TITLE
wrap outputs of example in try()

### PR DESCRIPTION
**Pull request by soerenmartius**

### CODE REVIEW

File: examples/ecr/outputs.tf [hunk 1]
```diff
@@ -5,17 +5,17 @@ output "repository" {
 
 output "registry_id" {
   description = "The registry ID where the repository was created."
-  value       = module.repository.repository.registry_id
+  value       = try(module.repository.repository.registry_id, null)
```
Comment (mariux) [inline examples/ecr/outputs.tf:8]: this should not be needed as `.registry_id` is always available... what is the idea?

Comment (mariux) [reply examples/ecr/outputs.tf:8]: same for the other `try()`s

Comment (soerenmartius) [reply examples/ecr/outputs.tf:8]: not if you have `module_enabled` set to `false`

Comment (mariux) [reply examples/ecr/outputs.tf:8]: there is no module_enabled in there..

Comment (soerenmartius) [reply examples/ecr/outputs.tf:8]: It's used in the underlying module. See

https://github.com/mineiros-io/terraform-aws-ecr/blob/master/outputs.tf#L3 and
https://github.com/mineiros-io/terraform-aws-ecr/blob/master/main.tf#L15

Comment (soerenmartius) [reply examples/ecr/outputs.tf:8]: Won't terraform return an error on the outputs since they would return null in case we set `module_enabled = false` ?

Comment (mariux) [reply examples/ecr/outputs.tf:8]: yes, but it is not set.... and it is also not settable...as the example module does not allow to set it... so it is not needed... ;)

**Pull request closed without merging**